tests/OSPSuite.HelpersForTests/ModelHelperForSpecs.cs

```diff
@@ -820,6 +822,25 @@ private ISpatialStructure getSpatialStructure()
          art.Add(newConstantParameter(ConstantsForSpecs.Q, 2));
          organism.Add(art);
 
+         var parameterReferencingNeighborhood = newConstantParameter("RefParam", 10);
+         var refFormula = _objectBaseFactory.Create<ExplicitFormula>()
+            .WithFormulaString("K*10")
+            .WithName("FormulaReferencingNBH");
+         parameterReferencingNeighborhood.Formula = refFormula;
+         //referencing parameter K between art_pls and bone_pls
+
+         var objectPath = _objectPathFactory.CreateFormulaUsablePathFrom(
```

Comment:
Here creating a path using the NBH keywords. looks like it's working :)

Reply:
looks good for me :)